uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -602,7 +623,7 @@ sub identify_scan_db {
 
 =pod
 
-=head3 insert_violated_scans($dbhr, $series_desc, $minc_location, ...)
+=head3 insert_violated_scans($dbhr, $series_desc, $minc_location, $patient_name, $candid, $pscid, $visit, $tr, $te, $ti, $slice_thickness, $xstep, $ystep, $zstep, $xspace, $yspace, $zspace, $time, $seriesUID)
```

Comment:
s line like THIS one is why it would have been nicer to keep the `...`!!!

Reply:
hehe. yep ;)